qualtran/bloqs/state_preparation/sparse_state_preparation_via_rotations.py

```diff
@@ -94,7 +100,13 @@ def from_sparse_array(
         coeffs: Union[Sequence[complex], NDArray[np.complex_], 'scipy.sparse.sparray'],
         phase_bitsize: SymbolicInt,
     ):
-        """Factory"""
+        """Factory to construct sparse state given the coefficients.
+
+        Args:
+            coeffs: A vector of coefficients `c_i`, either as a sequence/numpy array,
+                    or a scipy sparse array.
```

Comment:
extreme nit: we usually only indent the second line of a arg docstring one indent level (4 addtl spaces) instead of block-aligning them

Reply:
ah thanks! fixed